From a215baba639a3ce1ba9f7d158de7dfbec6e54ef1 Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Tue, 30 Oct 2018 17:03:37 +0200
Subject: [PATCH 748/767] net/phy: xgkr: Removed the unnecessary error message
 'No associated device tree node'

The error message 'No associated device tree node' issued from callback 'fsl_backplane_match_phy_device' was not necessary and annoying for other usecases without backplane, so it was removed.

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/phy/fsl_backplane.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/phy/fsl_backplane.c b/drivers/net/phy/fsl_backplane.c
index 46da9084a008..2a3458324b74 100644
--- a/drivers/net/phy/fsl_backplane.c
+++ b/drivers/net/phy/fsl_backplane.c
@@ -1659,7 +1659,6 @@ static int fsl_backplane_match_phy_device(struct phy_device *phydev)
 	const int num_ids = ARRAY_SIZE(phydev->c45_ids.device_ids);
 
 	if (!phydev->mdio.dev.of_node) {
-		dev_err(&phydev->mdio.dev, "No associated device tree node\n");
 		return 0;
 	}
 
-- 
2.17.0

